draft-pda-protocol.md

```diff
@@ -353,6 +353,8 @@ struct {
 enum { prio(0), hits(1) } PDAProto;
 
 opaque Url<1..2^16-1>;
+
+Duration uint64; /* Number of seconds elapsed between two instants */
```

Comment:
```suggestion
Time uint64; /* seconds elapsed since start of UNIX epoch */
Duration uint64; /* Number of seconds elapsed between two Time instants */
```